src/log_batch.rs

```diff
@@ -728,6 +728,14 @@ impl LogBatch {
 
     /// Adds a protobuf key value pair into the log batch.
     pub fn put_message<S: Message>(&mut self, region_id: u64, key: Vec<u8>, s: &S) -> Result<()> {
+        #[cfg(not(test))]
```

Comment:
No, this is not what I meant. I mean this:
```
#[not(cfg(test))]
fn is_internal_key(key, prefix) {
  if let Some(p) = prefix { ... }
  else {
    return is_internal_key(key, Some(ATOMIC_GROUP_KEY));
  }
}
#[not(cfg(test))]
fn make_internal_key(key, prefix) {
  assert(prefix == ATOMIC_GROUP_KEY);
  ...
}
```

Reply:
Got it